src/main/scala/com/fortysevendeg/android/sqlite/LogWrapper.scala

```diff
@@ -1,40 +1,66 @@
 package com.fortysevendeg.android.sqlite
 
+import Printer._
+
 trait LogWrapper {
 
-    val level: Int
+  def v(message: String)
+
+  def d(message: String)
 
-    def d(message: String)
+  def i(message: String)
 
-    def e(message: String)
+  def w(message: String)
 
-    def e(message: String, t: Throwable)
+  def e(message: String)
 
-    def i(message: String)
+  def e(message: String, t: Throwable)
```

Comment:
does it make sense if we re-define `e` method in this way?

def e(message: String, t: Option[Throwable] = None)? IMO, this Option could be justified :P